integration-test/helpers/mocks.js

```diff
@@ -31,5 +31,44 @@ export const constants = {
                 credential: '[data-ddg-inputtype="credentials.password"]'
             }
         }
+    },
+    featureToggles: {
```

Comment:
seems like these additions, `featureToggles` could drift from the true types - can we add some `@type`'s here to mitigate?